text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -31,4 +32,13 @@ target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/s
 target_compile_definitions(${TARGET_NAME} PRIVATE OPENVINO_TOKENIZERS_PATH=\"$<TARGET_FILE:openvino_tokenizers>\")
 set_target_properties(${TARGET_NAME} PROPERTIES CXX_STANDARD 11 CXX_STANDARD_REQUIRED ON)
 
+file(GLOB headers include/*.hpp)
+set_target_properties(${TARGET_NAME} PROPERTIES PUBLIC_HEADER "${headers}")
+
 target_link_libraries(${TARGET_NAME} PRIVATE openvino::runtime nlohmann_json::nlohmann_json)
+
+install(TARGETS ${TARGET_NAME}         
+        ARCHIVE DESTINATION lib
+        LIBRARY DESTINATION lib
+        RUNTIME DESTINATION bin 
```

Comment:
can we use https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html instead of hardcodes?